app/assets/stylesheets/members/index.css.scss

```diff
@@ -1,4 +1,7 @@
 .member {
+  @include breakpoint($mobile) {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets